core/domain/app_feedback_report_jobs_one_off_test.py

```diff
@@ -0,0 +1,242 @@
+# coding: utf-8
+#
+# Copyright 2020 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Tests for one-off jobs related to app feedback reports."""
+
+from __future__ import absolute_import  # pylint: disable=import-only-modules
+from __future__ import unicode_literals  # pylint: disable=import-only-modules
+
+import datetime
+
+from core.domain import app_feedback_report_jobs_one_off
+from core.platform import models
+from core.tests import test_utils
+import feconf
+
+(app_feedback_report_models,) = models.Registry.import_models([
+    models.NAMES.app_feedback_report])
+
+
+class ScrubReportsOneOffJobTests(test_utils.GenericTestBase):
```

Comment:
Other tests to consider adding:
- Running the job multiple times (shouldn't scrub anything a second time)
- Running the job a second time to scrub additional reports that were added after a previous successful scrub (to verify that the job will continue to scrub reports after the first run)